### ISSUE TITLE
Remove images from the context

### ISSUE THREAD

**Issue by sunithvs** [Sarahkhan20/GitZen#6]
File: docs/images/banner.png
============================================================
�PNG
<img width="1103" alt="image" src="https://github.com/user-attachments/assets/2dc015ff-0cd3-42d4-bbd9-6d20c1d8d0fe" />
This is causing the context to become large

**Comment by Sarahkhan20** [comment]
yes , that's true! I will word on removing certain  .jpg or .extension which doesn't contributes to the summary option. Thanks
